CONTRIBUTING.md

```diff
@@ -0,0 +1,58 @@
+#How to contribute
+Alfresco's maturity comes from the contributions of its community. Contributing back to Alfresco can be a bit confusing as Alfresco is a collection of separate open source projects. Each open source project officially sponsored by Alfresco has a page in this space detailing its governance policy and the method for reporting issues and submitting contributions.
+ 
+For the details on the Records Management project see[here.](https://community.alfresco.com/docs/DOC-6387-project-overview-records-management)
+
+##Getting started
+What you need to get started:
+
+* A [Jira](https://issues.alfresco.com/jira/projects/ALF/issues/?filter=allopenissues) account,
+
+* A [GitHub](https://github.com/) account
+
+Useful but not necessary:
+
+* An [Alfresco community](https://community.alfresco.com/) account - this is a good place to ask questions and find answers. 
+##Making changes
+
+We use Jira to track issues if you are committing a fix for a raised issue please include the ticket number in both the merge requests and the git commit messages.
+For example a fix for the ticket [ALF-21953](https://issues.alfresco.com/jira/browse/ALF-21953) will have the following git commit message: "ALF-21953: updated to use super pom v9"
+
+If you are Adding in a new feature or bug fix please do so [here.](https://issues.alfresco.com/jira/projects/ALF/issues/?filter=allopenissues) By raising a ticket in this project you will be agreeing to the Alfresco Contribution Agreement which can be found at the bottom of the 'Create Issue' form or alternatively attached to[this](https://community.alfresco.com/docs/DOC-7070-alfresco-contribution-agreement)page.
```

Comment:
Randomly capitalised "A" in "Adding".  Also - great that the contributor agreement is mentioned here - I missed it before.

Reply:
My bad, changing this now.